tests/functional/sign_in.js

```diff
@@ -115,8 +115,7 @@ define([
         .then(createUser(email, PASSWORD, { preVerified: true }))
         .then(fillOutSignIn(email, '  ' + PASSWORD))
         // success is seeing the error message.
-        .then(visibleByQSA('.error'))
-        .then(testElementTextInclude('.error', 'password'));
```

Comment:
Only minor thing is that before, this would test the error message was correctly talking about the password (instead of our wonderful "unexpected error" or similar). This could probably be kept, but just changing the test of the first argument to `'.tooltip'`.